docs/app/Examples/views/Advertisement/Types/AdvertisementExampleBanner.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Advertisement } from 'semantic-ui-react'
+
+const AdvertisementExampleBanner = () => (
+  <div>
+    <Advertisement unit='banner' test='Banner' />
+    <Advertisement unit='vertical banner' test='Vertical Banner' />
+    <Advertisement unit='top banner' test='Top Banner' />
+    <Advertisement unit='half banner' test='Half Banner' />
```

Comment:
I think the `<ComponentExample examplePath='views/Advertisement/Types/AdvertisementExampleBannerVertical' />` needs the `title` and `description` props. The doc site crashes when I try the code above